charts/yurt-app-manager/templates/admission-webhooks/job-patch/job-patchWebhook.yaml

```diff
@@ -0,0 +1,55 @@
+{{- if and .Values.admissionWebhooks.enabled .Values.admissionWebhooks.patch.enabled (not .Values.admissionWebhooks.certManager.enabled) }}
+apiVersion: batch/v1
+kind: Job
+metadata:
+  name:  {{ template "yurt-app-manager.fullname" . }}-admission-patch
+  namespace: {{ .Release.Namespace }}
+  annotations:
+    "helm.sh/hook": post-install,post-upgrade
+    "helm.sh/hook-delete-policy": before-hook-creation,hook-succeeded
+  labels:
+    app: {{ template "yurt-app-manager.name" . }}-admission-patch
+    {{- include "yurt-app-manager.labels" . | nindent 4 }}
+spec:
+  {{- if .Capabilities.APIVersions.Has "batch/v1alpha1" }}
+  # Alpha feature since k8s 1.12
+  ttlSecondsAfterFinished: 0
+  {{- end }}
+  template:
+    metadata:
+      name:  {{ template "yurt-app-manager.fullname" . }}-admission-patch
+      labels:
+        app: {{ template "yurt-app-manager.name" . }}-admission-patch
+        {{- include "yurt-app-manager.labels" . | nindent 8 }}
+    spec:
+      {{- with .Values.imagePullSecrets }}
+      imagePullSecrets:
+      {{- toYaml . | nindent 8 }}
+      {{- end }}
+      containers:
+        - name: patch
+          image: {{ .Values.imageRegistry }}/{{ .Values.admissionWebhooks.patch.image.repository }}:{{ .Values.admissionWebhooks.patch.image.tag }}
+          imagePullPolicy: {{ .Values.admissionWebhooks.patch.image.pullPolicy }}
+          args:
+            - patch
+            - --webhook-name={{ template "yurt-app-manager.fullname" . }}
+            - --namespace={{ .Release.Namespace }}
+            - --secret-name={{ template "yurt-app-manager.fullname" . }}-admission
+            - --patch-failure-policy={{ .Values.admissionWebhooks.failurePolicy }}
+            - --patch-mutating=true
+            - --log-level=trace
```

Comment:
maybe we need to add `--patch-validating=true` as well?

Reply:
The patch-validating default value is true.

Reply:
ok, i got it